content/en/docs/adopters.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Notary Project Adopters"
+description: "Known adopters who use Notation in production or integrate Notation into their products and services."
+type: docs
+weight: 9
+---
+
+The following organizations are known adopters who use Notation in production or integrate Notation into their products and services.
+
+We are happy and proud to have you all as part of the community! To add your use case and organization to this list, please raise a pull request.
+
+All organizations are sorted alphabetically below.
+
+| Organization | Website | Scenario | 
+| --- | --- | --- | 
+| Amazon | https://aws.amazon.com/ | [AWS Signer](https://docs.aws.amazon.com/signer/latest/developerguide/image-signing-prerequisites.html) |
+| containerd |  http://containerd.io/ |  [container image sign and verify with notation tool](https://github.com/containerd/nerdctl/blob/main/docs/notation.md) |
+| Harbor | https://goharbor.io/ |  Storage and display Notary Project signature |
```

Comment:
I found this link with Notation https://goharbor.io/docs/main/working-with-projects/working-with-images/sign-images/, but it contains information with cosign

Reply:
Updated